infra/modules/aws-host/variables.tf

```diff
@@ -49,6 +49,12 @@ variable "logs_kms_key_arn" {
   default     = null
 }
 
+variable "log_retention_days" {
```

Comment:
Ok, variable is passed but I don't see where it is set

Reply:
![SCR-20240321-oyyq](https://github.com/Worklytics/psoxy/assets/102831/816ef276-04ae-4c4d-bd3d-4adf35e5b18b)

Reply:
ah yeah sorry :facepalm: ; I was expecting to see the direct application on some AWS resource, but that was already there through `aws-psoxy-rest` and `aws-psoxy-bulk` so this was only providing the parameter instead of using the default